erpnext/startup/boot.py

```diff
@@ -75,3 +75,11 @@ def update_page_info(bootinfo):
 			"Sales Person Tree": {"title": "Sales Person Tree", "route": "Tree/Sales Person"},
 		}
 	)
+
+
+def bootinfo(bootinfo):
+	if bootinfo.get("user") and bootinfo["user"].get("name"):
+		bootinfo["user"]["employee"] = ""
+		employee = frappe.get_cached_value("Employee", {"user_id": bootinfo["user"]["name"]}, "name")
```

Comment:
It's better to do db.get_value(cache=True) here because redis cache doesn't work with filters.

Reply:
I don't think that will work either because it only seems to cache if filters is a string:

https://github.com/frappe/frappe/blob/fc5d946abbcd5ac7effa3683c57355e548e6f909/frappe/database/database.py#L559